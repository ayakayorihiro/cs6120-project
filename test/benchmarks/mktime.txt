1994 11 11 14 20 20