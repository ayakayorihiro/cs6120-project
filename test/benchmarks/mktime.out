You entered 1994 11 11 14 20 20
Got back (3988 06 02 00 40 40)
